examples/opensuse.yaml

```diff
@@ -3,7 +3,9 @@ images:
 # Hint: run `limactl prune` to invalidate the "Current" cache
 - location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-15.4-OpenStack-Cloud-Current.qcow2
   arch: "x86_64"
-# No aarch64 OpenStack build found :(
+  digest: "sha256:378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195"
```

Comment:
This digest has to be removed too

Reply:
Why, it seems to be correct:

```console
$ cat openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256
378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195 openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2
```

And I've successfully created an openSUSE instance with it:

```console
$ l start examples/opensuse.yaml
? Creating an instance "opensuse" Proceed with the current configuration
INFO[0000] Attempting to download the image              arch=x86_64 digest="sha256:378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195" location="https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-15.4-OpenStack-Cloud-Current.qcow2"
INFO[0001] Using cache "/Users/jan/Library/Caches/lima/download/by-url-sha256/27f3bb70bbc178edcae8edd360883ca9f2c9dc88c9d83cecd7d5b9752347ad05/data"
[...]
```

Reply:
The JeOS checksum seems unchanged across some mirrors:

```console
$ FILE=distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256
$ curl https://download.opensuse.org/$FILE
378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195 openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2
$ curl https://mirror.csclub.uwaterloo.ca/opensuse/$FILE
378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195 openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2
$ curl https:///mirror.its.dal.ca/opensuse/$FILE
378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195 openSUSE-Leap-15.4-JeOS.x86_64-OpenStack-Cloud.qcow2
```

However, there is something wrong with the checksums on openSUSE mirrors; I've noticed this for the Minimal image now:

```console
$ FILE=distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2.sha256
$ curl https://download.opensuse.org/$FILE
c686339ace6ab5d3cdf6999d00b3dec2b8adf11456f322d970d6caef74614381 openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2
$ curl https://mirror.csclub.uwaterloo.ca/opensuse/$FILE
07a770c25f26a1b298e862c6b24096f80e0e4c37938cffa7f9261854eba15e7a openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2
$ curl https://mirror.its.dal.ca/opensuse/$FILE
07a770c25f26a1b298e862c6b24096f80e0e4c37938cffa7f9261854eba15e7a openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2
```

So different mirrors have different images and different checksums:

```console
$ FILE=distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2
$ curl -so uwaterloo.qcow2 https://mirror.csclub.uwaterloo.ca/opensuse/$FILE
$ curl -so opensuse.qcow2 https://downloadcontent.opensuse.org./$FILE
$ sha256sum uwaterloo.qcow2
07a770c25f26a1b298e862c6b24096f80e0e4c37938cffa7f9261854eba15e7a  uwaterloo.qcow2
$ sha256sum opensuse.qcow2
c686339ace6ab5d3cdf6999d00b3dec2b8adf11456f322d970d6caef74614381  opensuse.qcow2
```

So at least the servers seem to be internally consistent, but I guess it means the images are being rebuilt with the same names, so these are not repeatable downloads and can't have a digest in the template.